fix: [M3-7170] - Create VPC Region and Label Validation (#9750)
linode/manager: packages/manager/.changeset/pr-9750-fixed-1696365803471.md

---
"@linode/manager": Fixed
---

> Ensure that error messages for region, label, and description on VPC Create flow do not incorrectly persist ([#9750](https://github.com/linode/manager/pull/9750))

Reviewer: ```suggestion
Persistent error messages for Region, Label, and Description fields in VPC Create flow ([#9750](https://github.com/linode/manager/pull/9750))
```
